src/main/java/com/slcube/shop/business/review/controller/ReviewController.java

```diff
@@ -0,0 +1,64 @@
+package com.slcube.shop.business.review.controller;
+
+import com.slcube.shop.business.member.dto.MemberSessionDto;
+import com.slcube.shop.business.review.dto.*;
+import com.slcube.shop.business.review.service.ReviewService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/api/reviews")
+@RequiredArgsConstructor
+public class ReviewController {
+
+    private final ReviewService reviewService;
+
+    @PostMapping
+    public ResponseEntity<Long> saveReview(@RequestBody @Valid ReviewSaveRequestDto requestDto, @AuthenticationPrincipal MemberSessionDto memberSessionDto) {
+        Long reviewId = reviewService.saveReview(requestDto, memberSessionDto);
+        return new ResponseEntity<>(reviewId, HttpStatus.CREATED);
+    }
```

Comment:
여기서는 uri를 반환하는게 어떨까요?? header에 location 이라는 걸로 uri 가 반환 되서요

```suggestion
    @PostMapping
    public ResponseEntity<Long> saveReview(@RequestBody @Valid ReviewSaveRequestDto requestDto, @AuthenticationPrincipal MemberSessionDto memberSessionDto) {
        Long reviewId = reviewService.saveReview(requestDto, memberSessionDto);
        URI userCreateUri = ServletUriComponentsBuilder.fromCurrentRequest()
                .path("/{id}")
                .buildAndExpand(reviewId)
                .toUri();
        return ResponseEntity.created(uri).build();
    }
```